tap_jira/streams.py

```diff
@@ -376,6 +373,9 @@ def sync_project(self, fieldNames, knownFields, project_key_or_id = None):
         last_updated = Context.update_start_date_bookmark(updated_bookmark)
         timezone = Context.retrieve_timezone()
         start_date = last_updated.astimezone(pytz.timezone(timezone)).strftime("%Y-%m-%d %H:%M")
+        if datetime.datetime(2024, 5, 17, 0, 0, tzinfo=pytz.utc) < last_updated < datetime.datetime(2024, 7, 23, 5, 0, 0, tzinfo=pytz.utc):
```

Comment:
These ranges are pretty liberal, want to ensure that we don't miss any data.  Its possible that we will reingest ~5 days of data once or twice depending on when orgs run.